### DISCUSSION TITLE
Gap between banner and dashboard content

### DISCUSSION THREAD

**Discussion by aignaciodc** [TfTHacker/DashboardPlusPlus#26] [Q&A]
Hi! I just set up dashboard++ on ipad (thanks to the Medium article about how to deal with hidden files!!) but have a maybe minor issue. There’s a huge gap between my banner, and the first section of text “Family”. I redid the process twice - first time I already had the banner, just added the dashboard. Second time I wiped it all clean and did the dashboard first (following from Sergio’s instructions) and the banner after. Both times, big gap. Can’t figure out what I’m doing wrong or if there’s any settings/options or coding I can change to fix it. Hoping for some guidance! Also noticed if I make the banner smaller (150 instead of 250) or use “gradient” instead of solid, I can always see words behind it (see pic) even in read mode. Is that normal? 
![IMG_3591](https://github.com/TfTHacker/DashboardPlusPlus/assets/77202094/e477f7c3-0158-4557-901a-2073a072e3d6)
![IMG_3592](https://github.com/TfTHacker/DashboardPlusPlus/assets/77202094/789c6dbe-1489-4e0d-8176-ffd8f3dd8a7e)
![IMG_3594](https://github.com/TfTHacker/DashboardPlusPlus/assets/77202094/29bea09b-4758-4300-99c5-720677286ba1)

**Comment by hannah803** [comment]
I've encountered the same issue and hope for a solution soon.

**Reply by aignaciodc** [reply]
I’ve wondered if it’s just leaving room for things like dataview graphs/charts, but it seems odd. Don’t want to mess too much with the code trying to change it myself.

**Comment by TfTHacker** [comment]
I think this is related to the banner plugin being broken. If you use BRAT, you can install the banners beta where I believe its working.

I also have a more "manual" approach I use for banners. https://tfthacker.com/banner-image

**Reply by aignaciodc** [reply]
Thank you. I'll try this.

**Reply by waseeqahmad** [reply]
This worked like a charm thank you.
